### ISSUE TITLE
TimePicker - Excess black space

### ISSUE THREAD

**Issue by barmargalit** [ant-design/ant-design#13256]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Version
3.10.8

### Environment
Windows 10, Chrome (Though not really environment related).

### Reproduction link
[https://codepen.io/barmargalit/pen/pZGMjO](https://codepen.io/barmargalit/pen/pZGMjO)

### Steps to reproduce
1) Create any simple TimePicker component.
2) go down all the way to the end of the time dropdown (on all fields hours, mins and secs) and see unwanted blank space.

### What is expected?
bottom line of drop-down should be either fixed, or at the length of the longest list (secs).

### What is actually happening?
Excess black space at the bottom of the list.

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/8953
